addons/im_livechat/static/src/core/public_web/discuss_app_model_patch.js

```diff
@@ -22,25 +37,62 @@ patch(DiscussApp.prototype, {
         });
         this.livechatLookingForHelpCategory = fields.One("DiscussAppCategory", {
             compute() {
+                if (!this.store.has_access_livechat) {
+                    return null;
+                }
                 return {
                     extraClass: "o-mail-DiscussSidebarCategory-livechatNeedHelp",
-                    hideWhenEmpty: true,
                     icon: "fa fa-exclamation-circle",
-                    id: `im_livechat.category_need_help`,
+                    id: `im_livechat.sidebar_category_need_help`,
                     name: _t("Looking for help"),
+                    openByDefault: false,
```

Comment:
I believe it should be open by default.
The discoverability of the feature is better this way.
Us having so many persons that will want to fold it is an exception.
Looking for help should be easily accessible out of the box.